va/validation-authority_test.go

```diff
@@ -140,9 +140,9 @@ func httpSrv(t *testing.T, token string) *httptest.Server {
 			t.Logf("HTTPSRV: Path = %s\n", r.URL.Path)
 
 			keyAuthz, _ := core.NewKeyAuthorization(currentToken, accountKey)
-			t.Logf("HTTPSRV: Key Authz = '%s%s'\n", keyAuthz.String(), "\\n \\t")
+			t.Logf("HTTPSRV: Key Authz = '%s%s'\n", keyAuthz.String(), "\\n\\r \\t")
```

Comment:
That's *kind of* a test for whitespace trimming, introduced in #1142. Probably a good candidate for some refactoring. :smile:

I can try to wrap that into a separate test if you feel like it's worth it.